coffee-bom/coffee-bom-project/pom.xml

```diff
@@ -310,6 +310,18 @@
                 <artifactId>coffee-deltaspike-data-extension-deployment</artifactId>
                 <version>${project.version}</version>
             </dependency>
+
+            <dependency>
+                <groupId>hu.icellmobilsoft.coffee</groupId>
+                <artifactId>coffee-module-etcd-extension-deployment</artifactId>
```

Comment:
coffee-module-etcd-extension mint runtime modul is szükséges lenne ha tartani szeretnénk a quarkus extension standardját.